### PR TITLE
Fix bug 1671123 (Test innodb.innodb-wl6445 is unstable)

### PR DESCRIPTION
**Pull request by laurynas-biveinis**
Replace the custom server restart sequences which use asynchronous
send_shutdown with the right include files.

http://jenkins.percona.com/job/percona-server-5.6-param/1760/

### CODE REVIEW

File: mysql-test/suite/innodb/t/innodb-wl6445.test [hunk 1]
```diff
@@ -13,22 +13,7 @@ INSERT INTO WL6445.t1 VALUES(0,0),(1,1),(2,2);
...
---source include/wait_until_disconnected.inc
-
---echo # Restart server.
---exec echo "restart:--innodb-read-only" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
-
---enable_reconnect
---source include/wait_until_connected_again.inc
---disable_reconnect
-
+--source include/restart_readonly_mysqld.inc
```
Comment (percona-ysorokin) [inline mysql-test/suite/innodb/t/innodb-wl6445.test]: missing `--let $restart_parameters=restart:--innodb-read-only`?

File: mysql-test/suite/innodb/t/innodb-wl6445.test [hunk 2]
```diff
@@ -80,22 +65,7 @@ DROP DATABASE WL6445;
...
---source include/wait_until_disconnected.inc
-
---echo # Restart server.
---exec echo "restart:" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
-
---enable_reconnect
---source include/wait_until_connected_again.inc
---disable_reconnect
-
+--source include/restart_mysqld.inc
```
Comment (percona-ysorokin) [inline mysql-test/suite/innodb/t/innodb-wl6445.test]: missing `--let $restart_parameters=`

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Review by robgolebiowski — APPROVED** [review]
LGTM

**Pull request merged**